scania-digital-design-system/sdds
# fix(dropdown): no default on dropdown

**Describe pull-request**  
Fix for default values on dropdown component.

**Solving issue**  
Fixes: [AB#2547](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2547)

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Dropdown
3. Check that all the options are pre-selected.

**Screenshots**  
-

**Additional context**  
-
